activerecord/test/cases/migration/pending_migrations_test.rb

```diff
@@ -92,25 +87,24 @@ def test_with_stdlib_logger
         end
 
         private
-          def assert_pending_migrations
-            # Do twice to test that the error continues to be raised.
-            2.times do
-              assert_raises ActiveRecord::PendingMigrationError do
-                CheckPending.new(proc { flunk }).call({})
-              end
+          def assert_pending_migrations(*expected_migrations)
+            error = assert_raises ActiveRecord::PendingMigrationError do
+              CheckPending.new(proc { flunk }).call({})
+            end
+
+            assert_includes error.message, "Migrations are pending."
+            expected_migrations.each do |migration|
+              assert_includes error.message, migration
             end
           end
 
           def assert_no_pending_migrations
             app = Minitest::Mock.new
             check_pending = CheckPending.new(app)
 
-            # Do twice to also test the cached result.
-            2.times do
-              app.expect :call, nil, [{}]
-              check_pending.call({})
-              app.verify
-            end
+            app.expect :call, nil, [{}]
+            check_pending.call({})
+            assert app.verify, "expected app.verify to return true, but it returned #{app.verify}"
```

Comment:
I think this error message won't be shown because [`verify`](https://docs.seattlerb.org/minitest/Minitest/Mock.html#method-i-verify) will raise `MockExpectationError` if it fails (and otherwise always returns `true`).